x/emissions/params.go

```diff
@@ -20,6 +20,12 @@ func DefaultParams() Params {
 		MinWeightCadence:            uint64(60 * 60),               // 1 hour in seconds
 		MaxInferenceRequestValidity: uint64(60 * 60 * 24 * 7 * 52), // 52 weeks approximately 1 year in seconds
 		MaxRequestCadence:           uint64(60 * 60 * 24 * 7 * 52), // 52 weeks approximately 1 year in seconds
+		Sharpness:                   uint64(20),                    // controls going from stake-weighted consensus at low values to majority vote of above-average stake holders at high values
+		BetaEntropy:                 float32(0.25),                 // controls resilience of reward payouts against copycat workers
+		DcoefAbs:                    float32(0.001),                // delta for numerical differentiation
+		LearningRate:                float32(0.05),                 // speed of gradient descent
+		MaxGradientThreshold:        float32(0.001),                // gradient descent stops when gradient falls below this
+		MinStakeFraction:            float32(0.5),                  // minimum fraction of stake that should be listened to when setting consensus listening coefficients
```

Comment:
Probably we want to use `cosmosMath.Dec` instead of `float32`.

Reply:
same thing for `uint64` -> `cosmosMath.Uint`

Reply:
down, sg. Feel free to apply if you get to it before me.